keyboards/mode/m256wh/rules.mk

```diff
@@ -0,0 +1,2 @@
+# Enter lower-power sleep mode when on the ChibiOS idle thread
+OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
```

Comment:
```suggestion
```

Reply:
Committed in [a8f58e1](https://github.com/qmk/qmk_firmware/pull/20502/commits/a8f58e1cd11132d878dcbc92c8dfd1c8fb4094ea)